wf-federated/Snakefile

```diff
@@ -0,0 +1,44 @@
+configfile: "/data/config.yaml"
+
+from snakemake.remote.S3 import RemoteProvider as S3RemoteProvider
+
+# any other remote storage is possible as well e.g. FTP
+# Remote storage has to be suported by Snakemake.
+# https://snakemake.readthedocs.io/en/stable/snakefiles/remote_files.html
+
+S3 = S3RemoteProvider(access_key_id=os.environ["ACCESS_KEY_ID"], 
+                      secret_access_key=os.environ["SECRET_ACCESS_KEY"], 
+                      endpoint_url="https://s3-elixir.cloud.e-infra.cz")
+
+# adjust to your own path
+outfile_s3 = "BUCKET/PATH/demo_outfile_result.txt"
+remote_sizes_file_s3 = "BUCKET/PATH/demo_sizes_file.txt"
+remote_out_file_s3 = "BUCKET/PATH/demo_logo.png"
```

Comment:
Any way we can use an env var for `BUCKET/PATH`? Again, it would be great if we could separate the config from the descriptor file.

Reply:
done